xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h

```diff
@@ -34,7 +34,6 @@ class CDVDOverlaySSA : public CDVDOverlay
   {
     replace = true;
     m_libass = libass;
-    libass->Acquire();
```

Comment:
Removing this results in the refcount reaching zero at shutdown.
Either we need to avoid the Acquire here, or add another Release to match.